internal/api/api.go

```diff
@@ -268,6 +272,7 @@ func (api idler) isJenkinsUnIdled(openshiftURL, openshiftToken, namespace string
 }
 
 func respondWithError(w http.ResponseWriter, status int, err error) {
+	w.Header().Set("Content-Type", "application/json")
 	log.Error(err)
 	w.WriteHeader(status)
```

Comment:
This must come before any w.Write methods. So move the WriteHeader before w.Header.Set(...)